primitives/blockchain/src/backend.rs

```diff
@@ -293,9 +240,8 @@ pub trait Backend<Block: BlockT>:
 		//
 		// FIXME #1558 only issue this warning when not on a dead fork
```

Comment:
Is this FIXME still relevant? Issue seems to be closed.

Reply:
Please see my comment above, probably can be removed if we don't have a clever way to distinguish a hash on a  dead for from a bug. Probably at this point we may just assume that we are dealing with a block on a dead fork